annotations/tasks.py

```diff
@@ -278,3 +293,69 @@ def handle_file_upload(request, form):
     if file_content != None:
         tokenized_content = tokenize(file_content)
         return save_text_instance(tokenized_content, text_title, date_created, is_public, user, uri)
+
+
+
+
+
+
+@shared_task
+def process_import_task(user, project_id ,file, part_of_id, action, name):
+    with open(file, 'r') as f:
```

Comment:
put a try catch around this. sometimes files can't be open or something else happens and celery just silently failed. you still want to update the task though, so that the user knows that something failed.